mempool.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+func getMempool() {
```

Comment:
you should rename this function to `main()` otherwise `go run mempool.go utils.go` will throw an error. go needs to know which function to run when you write `go run something`

Reply:
Done